Index.html

```diff
@@ -169,13 +169,17 @@ <h2 class="about">About me</h2>
   </section>
   <section class="page4">
     <p class="pfinal">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-  <form class="cuadrollenado"> 
-    <p class="fill">Full name</p>
-  <img class="fillbar"src="./images/fillbar.svg">
-    <p class="fill">Email address</p> 
-      <img class="fillbar"src="./images/fillbar.svg">
-         <textarea class="comments">Enter text here</textarea>
-</form>
+    
+    <form class="form" method="post"> 
+      <p class="data">Full name</p>
+    <input type="text" name="name" maxlength="30" placeholder="Your name here" required pattern="[A-Za-z0-9]+" required/>
+    <img class="fillbar"src="./images/fillbar.svg">
+    <p class="data">Email adress</p>
```

Comment:
- [ ] For Best practice of HTML semantic I recommend you  to use `label` tag for "Full Name" and "Email address" instead of using `p` tag, for more information you can refer [here](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form)